src/scss/header.scss

```diff
@@ -0,0 +1,54 @@
+input::-webkit-input-placeholder {
+	font-style: italic;
+	font-weight: 300;
+	color: #e6e6e6;
+}
+
+input::-moz-placeholder {
+	font-style: italic;
+	font-weight: 300;
+	color: #e6e6e6;
+}
+
+input::input-placeholder {
+	font-style: italic;
+	font-weight: 300;
+	color: #e6e6e6;
+}
+
+h1 {
```

Comment:
do not style for generic tag likes button, div, ....... because it will affect all other places.
Review all of the css here and change it.
Tip: use a class to implement.